docs/get_started.rst

```diff
@@ -1,17 +1,353 @@
-Get started
+Get Started
 ===========
 
+.. contents:: 
+    :depth: 2
+    :local:
+
+
 Install CumulusCI
 -----------------
 
+.. note:: These installation instructions assume some familiarity with entering commands into a terminal.
+    If that's completely new to you, we recommend starting with the `CumulusCI setup module <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup>`_ module on Trailhead to walk you through step by step.
+
+
+On macOS 
+^^^^^^^^
+Ensure you have `Homebrew <https://brew.sh/>`_ installed as it is a prerequisite for installing CumulusCI on macOS.
+
+
+Install via ``pipx`` 
+******************************
+Follow `installation instructions for pipx <https://pipxproject.github.io/pipx/installation/>`_.
+Once ``pipx`` is installed, we can install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished you can `verify your installation`_.
+
+
+
+On Linux
+^^^^^^^^
+
+
+
+Install via ``pipx``
+**************************
+We recommend installing CumulusCI using ``pipx``, which will make sure that CumulusCI and its dependencies are installed into their own Python environment that is separate from other Pytho software on your computer.
+Installation instruction for ``pipx`` can be found `here <https://pipxproject.github.io/pipx/installation/>`_.
+Once ``pipx`` is installed, we can install CumulusCI::
+
+    $ pipx install cumulusci
+
+Once finished you can `verify your installation`_.
+
+
+
+On Windows
+^^^^^^^^^^
+
+
+
+Install Python 3
+********************
+#. Go to the `Python downloads page <https://www.python.org/downloads/windows/>`_.
+#. Download the latest Python 3 release. Most users should select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it may depend on your particular computer setup.
```

Comment:
Let's point them to Python 3.8 rather than the latest version, and make sure the screenshot reflects that, until we've had a chance to test that everything works on 3.9